onnxruntime/core/common/string_utils.h

```diff
@@ -37,5 +38,32 @@ inline InlinedVector<std::string_view> SplitString(std::string_view string_to_sp
   return result;
 }
 
+/**
+ * Trim a string from start inplace.
+ * @param s The string to trim.
+ */
+inline void LeftTrimString(std::string& s) {
+  s.erase(s.begin(), std::find_if(s.begin(), s.end(), [](unsigned char ch) { return !std::isspace(ch); }));
```

Comment:
the function is to trim spcae from string? if so, why we need left trim and right trim both, left or right is enough?

can't see what the function is doing from function name.

Reply:
Fixed. Thanks!

Reply:
Oh the intention here is, just remove the space in the begging and end, not those ones in the middle. Because some symbolic dim really has space in its name. :)